src/main/java/duke/BH.java

```diff
@@ -0,0 +1,168 @@
+                } else if (s1.equals("list")) {
+                    System.out.println(ui.getLine() + this.getList() + ui.getLine());
+                } else if (s1.equals("mark")) {
+                    int index = Integer.parseInt(inputArray[1]) - 1;
+                    if (index < this.getListSize()) {
+                        System.out.println(ui.getLine() + "Well done! \n" + this.mark(index) + ui.getLine());
+                    } else {
+                        System.out.println("Index out of range");
+                    }
+                } else if (s1.equals("unmark")) {
+                    int index = Integer.parseInt(inputArray[1]) - 1;
+                    System.out.println("Oh no! \n" + this.unMark(index));
+                } else if (s1.equals("todo")) {
+                    String task = inputArray[1];
+                    Task newTask = new Todo(task);
+                    this.addToList(newTask);
+                    System.out.println(ui.getLine() + "Task added: " + newTask.toString() + "\n" +
+                            "Now you have " + this.getListSize() + " tasks in the list" + ui.getLine());
+                } else if (s1.equals("deadline")) {
+                    String s = inputArray[1];
+                    String[] arr = s.split("/by");
+                    String task = arr[0];
+                    String time = arr[1];
+                    Task newTask = new Deadline(task, time);
+                    this.addToList(newTask);
+                    System.out.println(ui.getLine() + "Task added:" + newTask.toString() + "\n" +
+                            "Now you have " + this.getListSize() + " tasks in the list" + ui.getLine());
+                } else if (s1.equals("event")) {
+                    String s = inputArray[1];
+                    String[] arr = s.split("/at");
+                    String task = arr[0];
+                    String time = arr[1];
+                    Task newTask = new Event(task, time);
+                    this.addToList(newTask);
+                    System.out.println(ui.getLine() + "Task added:" + newTask.toString() + "\n" +
+                            "Now you have " + this.getListSize() + " tasks in the list" + ui.getLine());
+                } else if (s1.equals("delete")) {
+                    int index = Integer.parseInt(inputArray[1]) - 1;
+                    System.out.println(ui.getLine() + "Okay, I have remove this task:\n" +
+                            this.deleteTask(index) + ui.getLine());
+                } else if (s1.equals("check")) {
+                    LocalDate date = LocalDate.parse(inputArray[1]);
+                    this.checkDate(date);
+                } else if (s1.equals("find")) {
+                    String[] arr = input.split(" ", 2);
+                    String word = arr[1];
+                    this.checkWord(word);
+                } else {
+                    System.out.println("Wrong input, please check again");
+                }
```

Comment:
Might want to consider a switch case instead of better readability.

Reply:
Sure I will try use enum in the next version